mws/mws_workspaces_test.go

```diff
@@ -258,6 +258,39 @@ func TestMwsAccGcpWorkspaces(t *testing.T) {
 	})
 }
 
+func TestMwsAccGcpWorkspacesWithExpectedProvisioning(t *testing.T) {
```

Comment:
Separately, two more test cases:
1. New workspace in provisioning state -> set expected state to RUNNING
2. New workspace in provisioning state -> unset expected state, workspace should progress to RUNNING.